examples/ioredis/README.md

```diff
@@ -0,0 +1,26 @@
+# ioredis Example
+
+An example of how to setup [ioredis](https://github.com/luin/ioredis) with Remix.
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/ioredis)
+
+## Usages
+
+- [Install](https://redis.io/topics/quickstart) or use your existing Redis server.
+- Duplicate the local `.env.example` file to `.env` and change the REDIS_URL environment variable to your redis server URL.
```

Comment:
What do you say about including a docker-compose file that starts a redis server? I do that in my app: https://github.com/kentcdodds/kentcdodds.com/blob/2af8efc6cebf25052f1f95f2210317a8086d74ef/docker-compose.yml#L3-L11

That way people can get redis running without having to install redis and configure themselves.

Reply:
I think we shouldn't add the docker-compose file to the example.

Because it only useful for people who are already using docker and don't have redis installed. 
And it may cause other people to get the file that are not irrelevant to them.

So I think we just add about it to README.md should be enough.